### DISCUSSION TITLE
编辑器一加静态碰撞器就点击预览显示不出来

### DISCUSSION THREAD

**Discussion by SKT1yang** [galacean/discussion#94] [Editor]
添加碰撞器组件后，点击预览项目报错：
<img width="971" alt="image" src="https://github.com/galacean/discussion/assets/34095649/c464c345-62e4-4cfa-ae02-498b4d8fd8a9">

需要将项目配置里的物理后端选择一下包，希望项目有个默认值

**Comment by gz65555** [comment]
代码已经合并：https://github.com/galacean/editor/pull/1788

**Comment by gz65555** [comment]
已经发布，目前默认物理后端是 PhysX
![image](https://github.com/galacean/discussion/assets/6926263/d1b9ddd4-3732-4b68-8532-7f5aea7589a9)
